python/ray/internal/internal_api.py

```diff
@@ -11,6 +11,24 @@ def global_gc():
     worker.core_worker.global_gc()
 
 
+def memory():
+    """Returns a formatted string describing memory usage in the cluster."""
+
+    import grpc
+    from ray.core.generated import node_manager_pb2
+    from ray.core.generated import node_manager_pb2_grpc
+
+    # We can ask any Raylet for the global memory info.
```

Comment:
I think it would be better to do this through the core worker to keep RPC clients there but not a big deal